### PR TITLE
feat: allow sort and filter forked session

### PR DESCRIPTION
**Pull request by darkskygit**

### CODE REVIEW

**Comment by darkskygit** [comment]
* **#7519** <a href="https://app.graphite.dev/github/pr/toeverything/AFFiNE/7519?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `canary`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @darkskygit and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: packages/backend/server/src/schema.gql [hunk 1]
```diff
@@ -554,8 +559,11 @@ type Query {
 
 input QueryChatHistoriesInput {
   action: Boolean
+  fork: Boolean
   limit: Int
+  messageOrder: [ChatHistoryOrder!]
```
Comment (forehalo) [inline packages/backend/server/src/schema.gql]: forget to regenerate type? why array here

Comment (darkskygit) [reply packages/backend/server/src/schema.gql]: fixed

Comment (forehalo) [reply packages/backend/server/src/schema.gql]: client as well

Comment (darkskygit) [reply packages/backend/server/src/schema.gql]: done

**Review by forehalo — APPROVED** [review]

**Pull request merged**